Makefile

```diff
@@ -29,12 +29,22 @@ clean-docs:
 	rm -rf apps/bondy_broker_bridge/doc/*
 	rm -f apps/bondy_broker_bridge/doc/.build
 
+CT_SUITE_FILE?=
+ifdef CT_SUITE_FILE
+CT_SUITE_ARGS = --suite ${CT_SUITE_FILE}
+else
+CT_SUITE_ARGS =
+endif
+
 test: xref
-	${REBAR} as test ct
+	${REBAR} as test ct ${CT_SUITE_ARGS}
 
 xref:
 	${REBAR} xref skip_deps=true
 
+cover: xref
```

Comment:
Again another _reminder_ for running the test and get the coverage on one file or on all.